src/main/java/com/github/manolo8/darkbot/gui/players/PlayerEditor.java

```diff
@@ -46,6 +85,9 @@ public void setup(Main main) {
 
         tagEditor.setup(main);
 
+        EventBrokerAPI eventBroker = main.pluginAPI.requireAPI(EventBrokerAPI.class);
+        eventBroker.registerListener(this);
```

Comment:
doing this on the setup method, means it will be called every time this is re-setup (this is done when changing config profile).
this means you will be registering the listener multiple times with a new dispatcher each time

you must take care of this